### PR TITLE
l10n_mk: Migrated account charts to version 9.0

### PR DESCRIPTION
**Pull request by DarkoNikolovski**

### CODE REVIEW

File: l10n_mk/__openerp__.py [hunk 1]
```diff
@@ -3,26 +3,26 @@
...
     ],
     'data': [
-        'data/account.account.type.csv',
-        'data/account.tax.code.template.csv',
-        'data/account.account.template.csv',
-        'data/l10n_mk_chart_template.xml',
-        'data/l10n_mk_wizard.xml',
-        'data/account.tax.template.csv',
-        'data/fiscal_position_template.xml',
+	'data/account.account.tag.csv',
```
Comment (kirca) [inline l10n_mk/__openerp__.py]: need to use spaces ;)

File: l10n_mk/__openerp__.py [hunk 2]
```diff
@@ -3,26 +3,26 @@
 # this module contains the full copyright notices and license terms.
 {
     'name': 'Macedonian localization',
-    'version': '1.0',
+    'version': '2.0',
     'author': 'HacBee UAB',
```
Comment (kirca) [inline l10n_mk/__openerp__.py]: Please change to HBEE

**Comment by kirca** [comment]
Please update the COPYRIGHT too like here https://github.com/HBEE/l10n-macedonia/commit/8a9a303cec93cf3124ee989d2b9398e25f76ec3f

**Comment by DarkoNikolovski** [comment]
Corrections are finished. Please check and if everything is fine we can merge  the branch.

**Comment by kirca** [comment]
Looks good, lets merge

**Pull request merged**